src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1361,6 +1368,7 @@ oop java_lang_Class::create_basic_type_mirror(const char* basic_type_name, Basic
   assert(static_oop_field_count(java_class) == 0, "should have been zeroed by allocation");
 #endif
   set_modifiers(java_class, JVM_ACC_ABSTRACT | JVM_ACC_FINAL | JVM_ACC_PUBLIC);
+  set_is_primitive(java_class);
```

Comment:
Just wondering what the comments at the start of this method are alluding to now that we do have a field at the Java level. ???